### PR TITLE
Updated for msal 1.0.0

### PR DESCRIPTION
**Pull request by workeffortwaste**

### CODE REVIEW

**Comment by workeffortwaste** [comment]
If you wanted to continue to use the live example you would need to re-add your own credentials.

File: src/services/auth.service.js [hunk 1]
```diff
@@ -40,14 +35,21 @@ export default class AuthService {
   logout() {
     this.app.logout();
   };
+  getAccount(){
+    if (this.app.getAccount()) {
+            return this.app.getAccount();
```
Comment (sunilbandla) [inline src/services/auth.service.js:40]: Can you please fix the formatting here?

File: src/App.vue [hunk 2]
```diff
@@ -35,6 +35,12 @@ export default {
     this.authService = new AuthService();
     this.graphService = new GraphService();
   },
+  mounted() {
+    let account = this.authService.getAccount();
+    if(account){
```
Comment (sunilbandla) [inline src/App.vue:40]: Can you please add space after `if`?

File: src/services/auth.service.js [hunk 3]
```diff
@@ -2,30 +2,25 @@ import * as Msal from 'msal';
 
 export default class AuthService {
   constructor() {
-    let PROD_REDIRECT_URI = 'https://sunilbandla.github.io/vue-msal-sample/';
-    let redirectUri = window.location.origin;
-    if (window.location.hostname !== '127.0.0.1') {
```
Comment (sunilbandla) [inline src/services/auth.service.js:7]: Does it work on localhost without this check?

**Review by sunilbandla — CHANGES_REQUESTED** [review]
Sorry about the late response. Thanks for the PR!

**Pull request closed without merging**